.github/workflows/translation_check.py

```diff
@@ -0,0 +1,174 @@
+    not_needed = dict.fromkeys(languages, 0)
+
+    for lang in languages:
+        translations = file_to_dict(dir_with_translations + '/' + lang + '.txt')
+
+        for base_string in en_gb:
+            if base_string not in translations:
+                missing_counters[lang] += 1
+                if print_info:
+                    print(f'[{lang}] Missing translation: {base_string}')
+            elif en_gb[base_string] == translations[base_string]:
+                same_counters[lang] += 1
+                if print_info:
+                    print(f'[{lang}] Same translation: {base_string}')
+
+        for translation in translations:
+            if translation not in en_gb:
+                not_needed[lang] += 1
+                if print_info:
+                    print(f'[{lang}] Unnecessary translation: {translation}')
+
+    result = {'missing': missing_counters, 'same': same_counters, 'not_needed': not_needed}
+    return result
+
+
+def format_result(count_on_master, count_on_pr):
+    """
+    Formats the result cell by formatting it bold if there is a difference between master and pr
+    """
+    missing = str(count_on_pr)
+    diff = count_on_pr - count_on_master
+    if diff != 0:
+        missing += ' ({0}{1})'.format('+' if diff > 0 else '', diff)
+        missing = f'**{missing}**'
+    return missing
+
+
+def prepare_spoiler(summary, details):
+    """
+    Prepares a foldable spoiler
+    """
+    return f'<p><details><summary>{summary}</summary>{details}</details></p>'
+
+
+def run():
+    """runs the script and outputs the result in result.md file"""
+    read_languages_from_master_and_pr()
+    prepare_translation_report()
+
+
+def prepare_translation_report():
+    master_branch = count_translations(MASTER_LANG_DIR, False)
+    pull_request = count_translations(PR_LANG_DIR, True)
+    missing = prepare_spoiler('Missing', 'The translation is not added to translation file. '
+                                         '(e.g. STR_9999 is in `en-GB` but is not available in given language)')
+    not_needed = prepare_spoiler('Not needed', 'The translation file contains entries that are not in `en-GB` '
+                                               'and should be removed '
+                                               '(e.g. STR_9999 exits in given language but is not in `en-GB`)')
+    same = prepare_spoiler('Same', 'The translation and source string is exactly the same.  '
+                                   '(e.g. STR_9999 is `Umbrella` in both `en-GB` and given language). '
+                                   'This may be desired in some cases (e.g. `April` is the same in English and German)')
+    table_header = "| |" + missing + " | " + not_needed + " |" + same + "|\n"
+    table_header += "|---|---|---|---|\n"
+    languages_changed = []
+    for lang in languages:
+        filename = lang + '.txt'
+        file_master = MASTER_LANG_DIR + '/' + filename
+        file_pr = MASTER_LANG_DIR + '/' + filename
```

Comment:
```suggestion
        file_master = os.path.join(MASTER_LANG_DIR, filename)
        file_pr = os.path.join(MASTER_LANG_DIR, filename)
```

Use `os.path.join` to make sure the proper file separator is picked (depending on the OS)